src/lib.rs

```diff
@@ -137,4 +143,16 @@ mod tests {
         assert!(works4().is_some());
         assert!(works5());
     }
+
+    extern crate std;
```

Comment:
Could this perhaps use `assert!` or something other than `println!` to avoid needing to pull this in?

Reply:
Oh, well i figured it was harmless for a test since the test suite itself already pulls in std, but sure, one moment.